utils_nlp/models/transformers/named_entity_recognition.py

```diff
@@ -1,2 +1,430 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License.
\ No newline at end of file
+# Licensed under the MIT License.
+
+import torch
+import logging
+import numpy as np
+
+from torch.utils.data import TensorDataset
+from cached_property import cached_property
+from collections import Iterable
+
+from transformers.modeling_bert import (
+    BERT_PRETRAINED_MODEL_ARCHIVE_MAP,
+    BertForTokenClassification
+)
+
+from utils_nlp.models.transformers.common import (
+    MAX_SEQ_LEN,
+    TOKENIZER_CLASS,
+    Transformer,
+    get_device,
+)
+
+TC_MODEL_CLASS = {k: BertForTokenClassification for k in BERT_PRETRAINED_MODEL_ARCHIVE_MAP}
+
+
+class TokenClassificationProcessor():
+    """
+    Process raw dataset for training and testing, and post-process prediction
```

Comment:
I believe this class doesn't cover post-processing?

Reply:
There is a "get_predicted_token_labels" function to post-process the raw predictions and get the token labels.

Reply:
That's in the token classifier class right? The Processor class is just for preprocessing, I think.

Reply:
I see. I must have put it in the wrong place. I originally plan to add the post processing function in the TokenClassificationProcessor class.